docs/Overview.md

```diff
@@ -0,0 +1,157 @@
+
+TuxedoJS solves these problems by wrapping `React` and `Flux` together into a single framework that abstracts away all of the boilerplate and complexity associated with getting a `Flux` application up and running and communicating with React.  Additionally, Tux Provides powerful augmented `React` Classes that are designed to handle different aspects of the app frontend such as managing state and listening to events from stores.  The TuxedoJS app architecture looks like the following:
+
+NEED IMAGE
+
+Let's dive in to what these different pieces actually represent.
+
+### The Modular Framework Structure
+In the below examples we are using the `require` syntax associated with `CommonJS` modules.  Of note is the fact that we are never writing the line `require('tux');`. That's because there is no global object to require for TuxedoJS.  Instead, Tux is broken up into individual modules that handle a particular aspect of your application.  What's the advantage of this?
+
+Firstly, it allows you to write semantic require statements that make it extremly clear what tux modules you need for your individual app components.
+
+Secondly, it means that, unlike with other popular frameworks, Tux never adds any more weight to your app than what you need for your purposes. For example, TuxedoJS comes prepackaged with beautiful animations. If you don't want to use them however, they never get loaded and thus never add any bytes to your Javascript files. Even better, if you want to use just one or some TuxedoJS prebuilt animations, simply require those animations and use them, no other animations will get bundled.  If you don't need Tux Architecture for an app, simply don't require it and it will never get loaded.
+
+```
+    var Actions = require('tux/Actions');
+    var Stores = require('tux/Stores');
+    var React = require('tux/React');
+    var Architecture = require('tux/Architecture');
+    var Fly = require('tux/Animations/Fly');
+    var FadeDownBig = require('tux/Animations/Fade/DownBig');
+```
+
+### TuxedoJS provides a novel approach to store dependencies that allows you to write out all of your store relationships in one file using a simple and declarative syntax.
+This turns the process of managing store relationships from frustrating and difficult to easy and self-documenting, allowing you and future engineers in your codebase to easily reason about your application architecture. TuxedoJS is the first framework we know of that allows you to write out your model dependencies in such a clean and simple syntax and we are extremely excited to use this syntax in building complex web apps.
```

Comment:
Recommend changing '...future engineers in your...' to '...future engineers using your codebase...'